pkg/cmd/pr/checkout/checkout.go

```diff
@@ -42,17 +48,32 @@ func NewCmdCheckout(f *cmdutil.Factory, runF func(*CheckoutOptions) error) *cobr
 		Config:     f.Config,
 		Remotes:    f.Remotes,
 		Branch:     f.Branch,
+		Prompter:   f.Prompter,
+		BaseRepo:   f.BaseRepo,
 	}
 
 	cmd := &cobra.Command{
-		Use:   "checkout {<number> | <url> | <branch>}",
+		Use:   "checkout [<number> | <url> | <branch>]",
 		Short: "Check out a pull request in git",
-		Args:  cmdutil.ExactArgs(1, "argument required"),
```

Comment:
💭 I think we will want to use `cobra.MaximumNArgs` ([examples](https://github.com/search?q=repo%3Acli%2Fcli+cobra.MaximumNArgs%28&type=code)) here:

```Go
Args:  cobra.MaximumNArgs(1),
```